Formula/colfer.rb

```diff
@@ -0,0 +1,17 @@
+class Colfer < Formula
+  desc "Schema compiler for binary data exchange"
+  homepage "https://github.com/pascaldekloe/colfer"
+  url "https://github.com/pascaldekloe/colfer/archive/v1.8.1.tar.gz"
+  sha256 "5d184c8a311543f26c957fff6cad3908b9b0a4d31e454bb7f254b300d004dca7"
+  license "CC0-1.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-o", bin/"colf", "./cmd/colf"
```

Comment:
```suggestion
    system "go", "build", *std_go_args, "-o", bin/"colf", "./cmd/colf"
```

Reply:
Seems like those flags make the test fail now.

Reply:
What is the failure?

Reply:
The failure magically disappeared. Maybe it was a hick-up on the build server.